packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -1390,3 +1341,47 @@ export function requestPostPaintCallback(callback: (time: number) => void) {
     localRequestAnimationFrame(time => callback(time));
   });
 }
+// -------------------
+//     Resources
+// -------------------
+
+export const supportsResources = true;
+
+export {isHostResourceType};
+export function isHostResourceInstance(
```

Comment:
Where is this exported to? Seems like it's only called in this file?

Reply:
must have been leftover after a refactor. i'll pull it